t/dsl/send_file.t

```diff
@@ -36,7 +36,7 @@ use File::Spec;
 
     get '/filehandle' => sub {
         open my $fh, "<:raw", __FILE__;
-        send_file( $fh, content_type => 'text/plain' );
+        send_file( $fh, content_type => 'text/plain', charset => 'utf-8' );
```

Comment:
Except.. not in this instance. There's no encoding going on here; the `charset` gets appends to the response `Content-Type` header. Happy to make that clearer in the Manual..